content/hardware/04.pro/shields/portenta-vision-shield/tutorials/camera-to-bitmap-sd-card/content.md

```diff
@@ -49,9 +49,16 @@ You will be using the **Himax HM-01B0 camera module** which has a resolution of
 Inside the sketch, you can use these libraries to access the camera APIs, also compatible with the [Arduino Nicla Vision](https://docs.arduino.cc/hardware/nicla-vision)
 ```cpp
 #include "camera.h" // Multi Media Card APIs
-#include "himax.h"  // API to read from the Himax camera found on the Portenta Vision Shield
+
+//For the Vision Shield Rev.1
+#include "himax.h"  // API to read from the Himax camera found on the Portenta Vision Shield Rev.1
+
+//For the Vision Shield Rev.2
+#include "hm0360.h" // API to read from the Himax camera found on the Portenta Vision Shield Rev.2

```

Comment:
in the rest of the examples version 1 library is commented, should we do it here too?

Reply:
On this specific case is not necessary since this is not part of the final sketch I am just mentioning the possible libraries to be used.

Reply:
as long as it is clear in the text I'm okay with that

let's move forward